components/accordion/index.css

```diff
@@ -621,19 +621,23 @@ governing permissions and limitations under the License.
 
 .spectrum-Accordion-item {
 	&.is-open {
-		> .spectrum-Accordion-itemHeading
-			> .spectrum-Accordion-itemIconContainer
-			> .spectrum-Accordion-itemIndicator {
-			/* stylelint-disable declaration-property-value-no-unknown */
-			transform: logical rotate(90deg);
-			/* stylelint-enable declaration-property-value-no-unknown */
+		> .spectrum-Accordion-itemHeading > .spectrum-Accordion-itemIconContainer > .spectrum-Accordion-itemIndicator {
+			transform: rotate(90deg);
+
+			[dir="rtl"] &,
+			&:dir(rtl) {
+				transform: matrix(-1, 0, 0, 1, 0, 0) rotate(90deg);
```

Comment:
We had initially [discussed](https://adobedesign.slack.com/archives/CS22GNRU2/p1702332156131419) removing this plugin and @jawinn mentioned that we might be able to get creative with "global" custom properties to help us get the value(s) we want out of these `transform` properties. Should we revisit that discussion?

Alternatively, if we don't go fully toward what was discussed there, perhaps we could at least pull the `matrix` part out of these places and house that in a custom property.

Reply:
I took a stab at a lower impact version of this approach.  I added a logical rotation definition to the root class for each impacted component, then leveraged that variable in the uses for the file.

```
[dir="rtl"] &,
&:dir(rtl) {
  --spectrum-logical-rotation: matrix(-1, 0, 0, 1, 0, 0);
}
```